tests/php/Command/DocumentationTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @copyright Copyright (c) 2023 Vitor Mattos <vitor@php.rio>
+ *
+ * @author Vitor Mattos <vitor@php.rio>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace OCA\Talk\Tests\php\Command;
+
+use OCP\App\IAppManager;
+use Test\TestCase;
+
+final class DocumentationTest extends TestCase {
+	public function testCommandIsDocumented(): void {
```

Comment:
Let's just use a github action which runs the command you added and checks that the diff is empty?

Reply:
By this way we only will identify that need to update the documentation when push and run the CI.

I updated the unit test to verify if the documentation is up to date and not only to check if the .md file have the command name.